_sass/_variables.scss

```diff
@@ -61,6 +61,7 @@ $light-black: #303030;
 
 // State pages
 $green-land: #d5e5cb;
+$chart-green: #a5d78a;
```

Comment:
Actually, nvm. It looks fine for Utah, maybe even more vibrant, but on other states where there is less federal land ownership, it starts looking overwhelmingly bright:

<img width="392" alt="screen shot 2016-06-07 at 5 12 28 pm" src="https://cloud.githubusercontent.com/assets/4803473/15876482/403690c0-2cd3-11e6-8a99-9096a8f33fd9.png">